test_settings.py.sample

```diff
@@ -1,2 +1,7 @@
-USERNAME = 'USERNAME'
-PRIVATE_KEY = 'PRIVATE_KEY'
+CONNECTION_OPTIONS = {
```

Comment:
I mean - we need to emphasize that user should change username and key, nothing more. Maybe do it like this in `..._sample.py`:
`username = 'REPLACE_WITH_YOUR_USERNAME'`